cosmosdb/client_endpoint.bal

```diff
@@ -173,14 +183,14 @@ public client class DataPlaneClient {
         return mapHeadersToResultType(response);
     }
 
-    # Query documents.
+    # Queries documents.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container to query
     # + sqlQuery - A string containing the SQL query
-    # + resourceQueryOptions - The `ResourceQueryOptions` which can be used to add additional capabilities to the
-    #                          request
-    # + return - If successful, returns a `stream<Document, error>`. Else returns `Error`.
+    # + resourceQueryOptions - The `cosmos_db:ResourceQueryOptions` which can be used to add additional capabilities 
+    #                          to the request
+    # + return - If successful, returns a `stream<cosmos_db:Document, error>`. Else returns `cosmos_db:Error`.
```

Comment:
Suggestion: "`stream<cosmos_db:Document, error>`, or else the `cosmos_db:Error` error"